src/cdk/text-field/autosize.ts

```diff
@@ -211,6 +236,16 @@ export class CdkTextareaAutosize implements AfterViewInit, DoCheck, OnDestroy {
     this._previousValue = value;
   }
 
+  /**
+   * Resets the textarea to it's original size
+   */
+  reset() {
+    if (this._initialHeight === undefined) {
```

Comment:
Can you add a comment explaining why we skip this if it's undefined?

Reply:
Added a comment for it to be more clear why this was added